### PR TITLE
CI scripts and files to build NuGet.VisualStudio.nupkg.

### PR DESCRIPTION
**Pull request by deepakaravindr**
CI scripts and files to build NuGet.VisualStudio.nupkg.

@yishaigalatzer @emgarten @alpaix

### CODE REVIEW

File: scripts/cibuild/PackNuGetVisualStudioPackage.ps1 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Write-Host "NuspecPath is $NuspecPath"
+
+$opts = , "pack"
+$opts += $NuspecPath
+$opts += "-Version", $Version
+$opts += "-BasePath", $VisualStudioBuildOutputWithConfiguration
+$opts += "-OutputDirectory", $OutputNupkgsFolder
+
+Write-Host "Running 'nuget.exe $opts'"
+& $NuGetExe $opts
```
Comment (emgarten) [inline scripts/cibuild/PackNuGetVisualStudioPackage.ps1:36]: Check the exit code and that the output nupkg exists

**Comment by emgarten** [comment]
Add validation so that this doesn't pass on the CI when pack fails, then :shipit:

File: src/NuGet.Clients/VisualStudio/VisualStudio.csproj [hunk 2]
```diff
@@ -23,6 +23,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>..\..\..\artifacts\VisualStudio\Debug\NuGet.VisualStudio.XML</DocumentationFile>
```
Comment (alpaix) [inline src/NuGet.Clients/VisualStudio/VisualStudio.csproj:26]: use $(EnlistmentRoot) instead of relative

**Comment by alpaix** [comment]
:shipit:

File: src/NuGet.Clients/VisualStudio/VisualStudio.csproj (hunk 2 above)
Comment (deepakaravindr) [inline src/NuGet.Clients/VisualStudio/VisualStudio.csproj:26]: Fixed.

This was auto-generated. Actually, this should simply be <DocumentationFile>$(OutputPath)\NuGet.VisualStudio.XML</DocumentationFile> and not relative EnlistmentRoot

Comment (alpaix) [inline src/NuGet.Clients/VisualStudio/VisualStudio.csproj:26]: agree.

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.Client/commit/445736bfd87ff1391f0830458fdfffd824486bb0

**Pull request closed without merging**